### ISSUE TITLE
Please use GitHub Releases

### ISSUE THREAD

**Issue by ctnkaan** [JedWatson/classnames#372]
Hello I've noticed that there are no releases for newer versions, there are only tags.

It would be great if there are notes on each new version release so people can know what is new or if they need to change something with their code when there is a major update.

I would love to help with the documentation for releases if there is an opening.

Cheers!

**Comment by jonkoops** [comment]
There are release notes in the [HISTORY file](https://github.com/JedWatson/classnames/blob/main/HISTORY.md)

**Comment by dcousens** [comment]
The project hasn't traditionally used GitHub releases, but I concur it is nice if you use GitHub's fine grained `watch` features.

Pull requests are welcomed to add that to our https://github.com/JedWatson/classnames/blob/main/.github/workflows/release.yml workflow

**Comment by jonkoops** [comment]
Agreed, we can do some work for this. If anyone would like to contribute, feel free to open a PR. Perhaps we should also rename the file to `CHANGELOG.md`, which most dependency bots (e.g. Dependabot, Renovate) support.

**Comment by ctnkaan** [comment]
I would love to contribute! Will look into this today after work.

**Comment by ctnkaan** [comment]
Hey @dcousens and @dcousens 

I've opened a PR about this issue. Feel free to give any improvement ideas whenever you have time 😁

**Comment by jonkoops** [comment]
I'll take a look @ctnkaan, sorry for the delay, I've been rather backlogged recently.
